src/plugins/table/table-designer.js

```diff
@@ -144,8 +143,8 @@ function TableDesigner({ content, onContentChange }) {
         className={classes}
         style={getDesignerCellStyle(designerCell)}
         >
-        <DebouncedInput
-          elementType={MarkdownInput}
+        <MarkdownInput
+          debounced
```

Comment:
😳 We had quite the conundrum here then.. With the double debouncing that you mentioned..

Reply:
Yes, classic case double-wrap :laughing: